### PR TITLE
Review fix for PR 16529 preprocess with PIL the full val dataset and save binary 

### PR DESCRIPTION
**Pull request by lidanqing-vv**
Fix reviews of PR [16529](https://github.com/PaddlePaddle/Paddle/pull/16529)
Review:
[unzip not inplace](https://github.com/PaddlePaddle/Paddle/pull/16529#discussion_r270301436 )  Done
[Print hard to understand](https://github.com/PaddlePaddle/Paddle/pull/16529#discussion_r270301561) Done
[Print logging every 500 images](https://github.com/PaddlePaddle/Paddle/pull/16529#discussion_r270301628)   Done
[Existing file integrity](https://github.com/PaddlePaddle/Paddle/pull/16529#discussion_r270291156)  Done

### CODE REVIEW

**Comment by lidanqing-vv** [comment]
@luotao1 I will use checksum and md5 for this completeness of file. I am working on it now

**Comment by luotao1** [comment]
@bingyanghuang Could you help review it?

File: paddle/fluid/inference/tests/api/full_ILSVRC2012_val_preprocess.py [hunk 1]
```diff
@@ -115,19 +115,19 @@ def download_unzip():
...
+    des_dir, source_dir = download_unzip()
+    file_list = os.path.join(source_dir, 'val_list.txt')
+    output_file = os.path.join(des_dir, 'int8_full_val.bin')
     with open(file_list) as flist:
         lines = [line.strip() for line in flist]
         num_images = len(lines)
         if not os.path.exists(output_file):
             print(
-                'Preprocessing to binary file...<num_images><all images><all labels>...\n'
+                'Start to convert jpg to binary file, the file will be in form [num_images | all images in binary | all labels in binary]...\n'
```
Comment (bingyanghuang) [inline paddle/fluid/inference/tests/api/full_ILSVRC2012_val_preprocess.py]: The log info is too long and too confused. Maybe you can change it to "Start to convert jpg to binary file, the file will contain images and labels data"

Comment (lidanqing-vv) [reply paddle/fluid/inference/tests/api/full_ILSVRC2012_val_preprocess.py]: OK.

Comment (bingyanghuang) [reply paddle/fluid/inference/tests/api/full_ILSVRC2012_val_preprocess.py]: Please add "Start download Full ImageNet Validation dataset" before starting download the dataset, remove the print on line 109 and line 114.

**Comment by lidanqing-vv** [comment]
New PR [16608](https://github.com/PaddlePaddle/Paddle/pull/16608)

**Pull request closed without merging**